src/folders.ts

```diff
@@ -85,7 +85,7 @@ export class CMakeToolsFolderController implements vscode.Disposable {
   get(ws: vscode.WorkspaceFolder | Array<string> | undefined): CMakeToolsFolder | undefined {
     if (ws) {
       if (util.isArrayOfString(ws)) {
-        return this._instances.get(ws[-1]);
+        return this._instances.get(ws.slice(-1)[0]);
```

Comment:
Can it just be `ws[ws.length-1]`? Do we need to copy the reference to the last element?